Tests/KituraSampleRouterTests/KituraSampleTests.swift

```diff
@@ -198,7 +198,7 @@ class KituraSampleTests: KituraTest {
 
         runGetResponseTest(path: path) { body in
             do {
-                #if os(Linux)
+                #if os(Linux) && !swift(>=3.1)
```

Comment:
I like this. It hadn't occurred to me that I could negate that check.  I tried `swift(<3.1)` and that wouldn't compile, so I resigned myself to having to do `#if`/`#else`. Thanks.